device-types/Ubiquiti/U6-Mesh.yaml

```diff
@@ -0,0 +1,21 @@
+---
+manufacturer: Ubiquiti
+model: Access Point U6 Mesh
+part_number: U6-Mesh
+slug: ubiquiti-u6-mesh
+u_height: 0
+is_full_depth: false
+comments: |
+  [Access Point U6 Mesh](https://store.ui.com/collections/unifi-network-wireless/products/access-point-wifi-6-mesh)
+airflow: passive
+weight: 400
+weight_unit: g
+interfaces:
+  - name: eth0
+    type: 1000base-t
+    poe_mode: pd
+    poe_type: type2-ieee802.3at
+  - name: wlan0
+    type: ieee802.11n
```

Comment:
Where are you getting `n` from? This is an ax access point

Reply:
```suggestion
    type: ieee802.11x
```